Modules/_datetimemodule.c

```diff
@@ -6912,27 +6979,81 @@ _datetime_exec(PyObject *module)
     return -1;
 }
 
+static int
+module_traverse(PyObject *module, visitproc visit, void *arg)
+{
+    datetime_state *state = get_module_state(module);
+    Py_VISIT(state->us_per_ms);
+    Py_VISIT(state->us_per_second);
+    Py_VISIT(state->us_per_minute);
+    Py_VISIT(state->us_per_hour);
+    Py_VISIT(state->us_per_day);
+    Py_VISIT(state->us_per_week);
+    Py_VISIT(state->seconds_per_day);
+    Py_VISIT(state->strptime);
```

Comment:
nitpick: please respect structure member order, move it 2 lines below. Same for clear function.